src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const img = document.querySelector('.gallery__list');
+const largeImg = document.querySelector('.gallery__large-img');
+
+img.addEventListener('click', (events) => {
+  const targeted = events.target.closest('.list-item');
+
+  if (!targeted) {
+    return;
+  }
+
+  events.preventDefault();
+
+  largeImg.src = targeted.querySelector('a').href;
```

Comment:
The line `largeImg.src = targeted.querySelector('a').href;` assumes that each `.list-item` contains an anchor (`<a>`) element. Ensure that this structure is consistent in your HTML. If the anchor tag is not present or structured differently, this line may cause an error.